configure.ac

```diff
@@ -1158,6 +1158,36 @@ AC_MSG_RESULT(${enable_linker_map})
 AM_CONDITIONAL([OPENTHREAD_ENABLE_LINKER_MAP], [test "${enable_linker_map}" = "yes"])
 
 #
+# Dynamic MPL Interval
+#
+
+AC_ARG_ENABLE(dynamic_mpl,
```

Comment:
I would prefer to have this as a config in [src/core/openthread-core-default-config.h](https://github.com/openthread/openthread/blob/master/src/core/openthread-core-default-config.h).

Reply:
Updated, thanks.